client/components/SomeComponent/index.tsx

```diff
@@ -1,32 +1,25 @@
 import * as React from 'react';
 import Button from '@material-ui/core/Button';
 import { Grid } from '@material-ui/core';
-import styled from 'styled-components';
-import { withTheme } from '@material-ui/core/styles';
+// import styled from 'styled-components';
```

Comment:
```suggestion
```

Remove commented-out code?

Reply:
This PR should have been a draft, it's not done yet